### PR TITLE
NAS-105500 / 12.0 / Nas 105500

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/api-keys.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    reset: {
+        placeholder: T('reset'),
+        tooltip: T(''),
+    },
+
+    apikeyCopyDialog: {
+        title: T('API Key'),
+        save_button: T('COPY TO CLICKBOARD'),
+        close_button: T('CLOSE'),
+        api_key_warning: T('<b>Success!</b> The API key has been created/reseted. <b>It will only appear here once.</b>'),
```
Comment (Mrt134) [inline src/app/helptext/api-keys.ts:36]:
```suggestion
        api_key_warning: T('<b>Success!</b> The API key has been created or reset. <b>This is the only time the key is shown.</b>'),
```

File: src/app/helptext/api-keys.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+
+    reset: {
+        placeholder: T('reset'),
+        tooltip: T(''),
+    },
+
+    apikeyCopyDialog: {
+        title: T('API Key'),
+        save_button: T('COPY TO CLICKBOARD'),
```
Comment (Mrt134) [inline src/app/helptext/api-keys.ts:34]:
```suggestion
        save_button: T('COPY TO CLIPBOARD'),
```

File: src/app/helptext/api-keys.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { T } from '../translate-marker';
+
+export default {
+    title: T('API Keys'),
+    route_add_tooltip: T('Add API Key'),
+    col_name: T('Name'),
+    col_created_at: T('Created Date'),
+    deleteMsg_title: T('APK Key'),
```
Comment (Mrt134) [inline src/app/helptext/api-keys.ts:8]:
```suggestion
    deleteMsg_title: T('API Key'),
```

File: src/app/helptext/api-keys.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    formDialog: {
+        add_title: T('Add API Key'),
+        edit_title: T('Edit API Key'),
+        add_button: T('ADD'),
+        edit_button: T('SAVE'),
+    },
+
+    name: {
+        placeholder: T('Name'),
+        tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/api-keys.ts:24]:
```suggestion
        tooltip: T('Descriptive identifier for this API key.'),
```

File: src/app/helptext/api-keys.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    },
+
+    reset: {
+        placeholder: T('reset'),
+        tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/api-keys.ts:29]:
```suggestion
        tooltip: T('Remove the existing API key and generate a new random key. \
 A dialog shows the new key and has an option to copy the key. Back up and \
 secure the API key! The key string is displayed only one time, at creation.'),
```

**Comment by Mrt134** [comment]
FYI, I attempted to build test with yarn on a FreeNAS mini with 12.0 nightly from 3/25 installed. This error appears after selecting the API Keys menu option:
Screenshot_2020-04-03 TrueNAS - localhost

**Comment by lola27** [comment]
Thanks @Mrt134. there is a newer build from Jenkins.

**Pull request merged**